extensions/ARB/ARB_draw_elements_base_vertex.txt

```diff
@@ -120,16 +120,16 @@ Overview
 New Procedures and Functions
 
     void DrawElementsBaseVertex(enum mode, sizei count, enum type,
-         void *indices, int basevertex);
+         const void *indices, int basevertex);
 
     void DrawRangeElementsBaseVertex(enum mode, uint start, uint end,
-         sizei count, enum type, void *indices, int basevertex);
+         sizei count, enum type, const void *indices, int basevertex);
 
     void DrawElementsInstancedBaseVertex(enum mode, sizei count,
-         enum type, const void *indices, sizei primcount, int basevertex);
+         enum type, const void *indices, sizei drawcount, int basevertex);
```

Comment:
This one is actually supposed to be "instancecount"

Reply:
Fair enough, I'll look at the specification too.

Reply:
Done.